lib/tags.ts

```diff
@@ -6,9 +6,21 @@ import { Construct } from 'constructs'
  * Available Backup plans. Availability depends on your agreement with Enfo
  */
 export enum BackupPlan {
+  /**
+   * Creates backups in the region of the resource
+   */
   STANDARD = 'Standard',
+  /**
+   * Creates backups in eu-north-1, the Stockholm region
```

Comment:
This should be "Creates backups in the region of the resource, and copies of those backups in eu-north-1, the Stockholm region"